docs/components/miranum-ide/miranum-deployment.md

```diff
@@ -0,0 +1,202 @@
+                        config: 'http://localhost:9001/rest/deployment'
+                    test:
+                        bpmn: 'http://localhost:9002/rest/deployment'
+                        dmn: 'http://localhost:9003/rest/deployment'
+                        form: 'http://localhost:9004/rest/deployment'
+                        config: 'http://localhost:9005/rest/deployment'
+```
+
+#### Miranum Deployment Server Embedded
+
+In case you have a single spring application you can use the Miranum Deployment Server Embedded starter.
+It provides the Miranum Deployment Server as well as a build in Miranum Deployment Receiver implementation.
+
+> A full example is available in the [miranum-deployment-example](https://github.com/Miragon/miranum-ide/tree/main/spring-boot-apps/miranum-deployment/miranum-deployment-example) module.
+
+##### Usage
+
+1. Add the Miranum Deployment Server Embedded starter to your spring application.
+
+```xml
+
+<dependency>
+    <groupId>io.miragon.miranum</groupId>
+    <artifactId>miranum-deployment-server-embedded-starter</artifactId>
+    <version>${project.version}</version>
+</dependency>
+```
+
+2. Implement the `MiranumDeploymentReceiver` interface and provide it as spring bean.
+
+```java
+import io.miragon.miranum.deploymentreceiver.application.ports.out.MiranumDeploymentReceiver;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class ExampleDeploymentReceiver implements MiranumDeploymentReceiver {
+
+    @Override
+    public void deploy(final Deployment deployment, final List<String> tags) {
+        // TODO: Implement your deployment logic here
+    }
+
+}
+```
+
+### Miranum Deployment Receiver
+
+Add the Miranum Deployment Receiver REST starter to each spring application that should deploy process artifacts.
+In the Miranum Deployment Server you can configure the target applications via the application.yml file.
+
+> A full example is available in the [miranum-deployment-example](https://github.com/Miragon/miranum-ide/tree/main/spring-boot-apps/miranum-deployment/miranum-deployment-example) module.
+
+#### Usage
+
+1. Add the Miranum Deployment Receiver REST starter to your spring application.
+
+```xml
+
+<dependency>
+    <groupId>io.miragon.miranum</groupId>
+    <artifactId>miranum-deployment-receiver-rest-starter</artifactId>
+    <version>${project.version}</version>
+</dependency>
+```
+
+2. Implement the `MiranumDeploymentReceiver` interface and provide it as spring bean.
+
+```java
+import io.miragon.miranum.deploymentreceiver.application.ports.out.MiranumDeploymentReceiver;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class ExampleDeploymentReceiver implements MiranumDeploymentReceiver {
+
+    @Override
+    public void deploy(final Deployment deployment, final List<String> tags) {
+        // TODO: Implement your deployment logic here
+    }
+
+}
+```
+
+## Java 11 and Spring Boot 2 support
```

Comment:
💭 - I would move this to the top of the documentation page since this is kind of a prerequisite. 🤔 What is your opinion on this?

Reply:
👍🏻